### ISSUE TITLE
ppport.h missing

### ISSUE THREAD

**Issue by eserte** [PDLPorters/pdla-rest#17]
On some of my smokers compilation of PDLA-Rest-2.016000 fails, probably because ppport.h is not bundled:
```
...
cc -c    -DHAS_FPSETMASK -DHAS_FLOATINGPOINT_H -fno-strict-aliasing -pipe -fstack-protector -I/usr/local/include -O    -DVERSION=\"2.016000\"  -DXS_VERSION=\"2.016000\" -DPIC -fPIC "-I/usr/perl5.18.4p/lib/5.18.4/amd64-freebsd/CORE"   Window.c
Window.xs:12:10: fatal error: 'ppport.h' file not found
#include "ppport.h"   /* for backwards comaptibility */
         ^
1 error generated.
*** Error code 1
```

**Comment by mohawk2** [comment]
Hi Slaven, thanks for the report! It looks like 2.019100 is passing Ok on one of your FreeBSD boxes now (albeit 10.1, not the 13 that the above failure happened on).

Does that match what you're seeing?

**Comment by eserte** [comment]
Unfortunately I still see the problem with PDLA-Rest-2.019101

**Comment by mohawk2** [comment]
An example is http://www.cpantesters.org/cpan/report/7f248d60-6410-11e9-8125-1c8d1f24ea8f

**Comment by mohawk2** [comment]
I moved `ppport.h` from the zapped `Basic/Core` in that repo, but didn't update the `MANIFEST` because my MANIFEST-checker test was inadequate. I've updated both the test and the MANIFEST. Thanks!

**Comment by mohawk2** [comment]
Released as 2.019002.
